DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/mmhost/CMakeLists.txt

```
@@ -0,0 +1,311 @@
+# Direct CMake to use icpx rather than the default C++ compiler/linker on Linux
+# and icx-cl on Windows
+if(UNIX)
+    set(CMAKE_CXX_COMPILER icpx)
+else() # Windows
+    include (CMakeForceCompiler)
+    CMAKE_FORCE_CXX_COMPILER (icx-cl IntelDPCPP)
+    include (Platform/Windows-Clang)
+endif()
+
+cmake_minimum_required (VERSION 3.7.2)
+
+project(component_interfaces CXX)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+
+###############################################################################
+### Customize these build variables
+###############################################################################
+if(${TYPE} MATCHES PART1)
+    set(SOURCE_FILES part1_pointers/src/mmhost.cpp)
+elseif(${TYPE} MATCHES PART2)
+    set(SOURCE_FILES part2_single_host/src/mmhost.cpp)
+elseif(${TYPE} MATCHES PART3)
+    set(SOURCE_FILES part3_hosts/src/mmhost.cpp)
+elseif(${TYPE} MATCHES PART4)
+    set(SOURCE_FILES part4_ddr_hosts/src/mmhost.cpp)
+else()
+    message(FATAL_ERROR "No matching compile type found for ${TYPE}\n
+Valid types -DTYPE=[ PART1 / PART2 / PART3 / PART4 ]")
+endif()
+set(TARGET_NAME mmhost)
+
+# Use cmake -DFPGA_DEVICE=<device_family> to choose a
+# different device family (E.g. "Arria10" or "Stratix10") or a
+# specific part number (E.g. "10AS066N3F40E2SG") to generate a standalone IP.
+if(NOT DEFINED FPGA_DEVICE)
+    set(FPGA_DEVICE "Agilex7")
+else()
+    message(STATUS "Configuring the design with the following target: ${FPGA_DEVICE}")
+
+    # Check if the target is a BSP
+    if(IS_BSP MATCHES "1" OR FPGA_DEVICE MATCHES ".*pac_a10.*|.*pac_s10.*")
```

```suggestion
    if(IS_BSP MATCHES "1")
```
The PAC boards are no longer supported and I'm making changes to remove all mentions to them